prophyc/tests/test_prophyc.py

```diff
@@ -700,3 +703,32 @@ def test_model_evaluation_warnings(call_prophyc, tmpdir_cwd):
 prophyc: warning: X::y array has unknown size "UNKNOWN"
 prophyc: error: X byte size unknown
 """
+
+
+@pytest.mark.xfail(sys.version_info >= (3,), reason="implicit relative imports are gone in py3; #25", strict=True)
```

Comment:
Let's fix the problem by adding `.` in import statements in generated code and close #25 in favor of this.

Reply:
As discussed offline, I added a minimal change that let's py3 users use packages (nothing should change for py2).